src/lib/control_allocation/control_allocation/CMakeLists.txt

```diff
@@ -34,6 +34,8 @@
 px4_add_library(ControlAllocation
 	ControlAllocation.cpp
 	ControlAllocation.hpp
+	ControlAllocationMetric.cpp
+	ControlAllocationMetric.hpp
```

Comment:
These should be removed given that we have the `ControlAllocation` being able to do metric allocation now?